napari/conftest.py

```diff
@@ -671,8 +671,8 @@ def _check_throttle_info(path):
             return (
                 path
                 + " it's possible that there was a problem with unfinished work by a "
```

Comment:
why did these get left with double-quote but the next lines were changed?

Reply:
Because of `it's` that contains single quite in it.